src/Layout/toolset-tasks/toolset-tasks.csproj

```diff
@@ -1,13 +1,17 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <TargetFrameworks>$(SdkTargetFramework);net472</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">$(SdkTargetFramework)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\Cli\Microsoft.DotNet.Cli.Utils\RuntimeEnvironment.cs" />
```

Comment:
This is not great. Directly source reference a file to production code. The build tasks have been moving around several times.

Reply:
I don't know of a better way to do this. Both the production code, and this build task need this functionality. Do you have a recommendation?

> The build tasks have been moving around several times.

If it moves again, then this line will need to be updated. But the mover will get a compile error when they move it, so I don't see a problem.

Reply:
This is now removed with the latest changes.